docs/processing/preprocessing.md

```diff
@@ -49,38 +52,70 @@ We avert that session-wise *fMRIPrep*'s processes run into race conditions by pr
 
 ## Visualizing *fMRIPrep*'s individual reports
 
-Following our protocols<sup>[1]</sup>, the quality of unprocessed images MUST be assessed before and after preprocessing to verify it did not go awry.
+Following our protocols<sup>[1]</sup>, the quality of unprocessed images MUST be assessed [before](../data-management/mriqc.md#visualizing-mriqcs-individual-reports) and after preprocessing to verify it did not go awry.
 
-- [ ] Open each *fMRIPrep* report on a current Web Browser (*Google Chrome* is preferred).
-- [ ] Visualize the first section entitled `Summary` and apply the [exclusion criteria](qaqc-criteria-preprocessed.md#summary).
+### Anatomical preprocessing assessment
+
+- [ ] Open the *fMRIPrep* anatomical report on a current Web Browser (*Google Chrome* is preferred).
+- [ ] Assess the "Summary" section and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#summary).
+- [ ] Assess the "Anatomical conformation" section and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#anatomical-conformation).
+- [ ] Assess the mosaic showing the calculated brain mask and brain tissue segmentation, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#brain-mask-and-brain-tissue-segmentation-of-the-t1w).
+- [ ] Visualize the spatial normalization flickering mosaic, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#spatial-normalization-of-the-anatomical-t1w-reference).
+    Flickering between the subject and the template space is active while hovering your mouse on the mosaic area.
+- [ ] Assess the surface reconstruction mosaic, and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#surface-reconstruction).
+- [ ] Visualize the first section entitled `Summary` and apply the [QA/QC criteria](qaqc-criteria-preprocessed.md#summary).
 
     !!! warning "Running through the visual report does not have to be executed in a fixed order"
 
-        While we describe the run through the report as linear, often inspecting other reportlets can help make a decision about the exclusion criteria related to the reportlet at hand. 
+        While we describe the run through the report as linear, often inspecting other reportlets can help make a decision about the exclusion criteria related 
+        to the reportlet at hand. 
```

Comment:
linebreak
```suggestion
        While we describe the run through the report as linear, often inspecting other reportlets can help make a decision about the exclusion criteria related to the reportlet at hand. 
```

Please update to follow the directions we have sketched in the previous PR to qc of unprocessed data (there's a similar admonition that I have edited for clarity).